pkgs/top-level/all-packages.nix

```diff
@@ -29266,7 +29266,10 @@ with pkgs;
 
   keepassx = callPackage ../applications/misc/keepassx { };
   keepassx2 = callPackage ../applications/misc/keepassx/2.0.nix { };
-  keepassxc = libsForQt5.callPackage ../applications/misc/keepassx/community.nix { };
+  keepassxc = libsForQt5.callPackage ../applications/misc/keepassx/community.nix {
+    inherit (darwin.apple_sdk_11_0.frameworks) LocalAuthentication;
```

Comment:
This will probably go out of sync once SDK 11.0 is outdated, but I don't know any other solution

Reply:
@turion , agreed. Probably a repository-wide change will be made when a new SDK will come along, since multiple packages are using the same pattern. I'm not aware of any other solution than using SDK 11 to avoid the build errors.

Reply:
Fair enough.